docs/wiki/research/roadmap.md

```diff
@@ -17,43 +17,44 @@ The discussion, resources and all research and development on the core protocol
-| :----------------------------------- | :-------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------- |
-| Launch the Beacon Chain              |             A crucial step in Ethereum's shift to a proof of stake consensus mechanism              | Beacon Chain becomes the engine of block production, replacing mining. Validators have the role and responsibility for processing the validity of all transactions and proposing blocks | shipped </br> EIP-2982[^1] |
-| Merge Execution and Consensus Layers |              Ethereum's execution layer merged with the Beacon chain (consensus layer)              |                                             Proof of work activities ceased and the network's consensus mechanism shifted to proof of stake                                             | shipped                    |
-| Enable Withdrawals                   | The last of the three-part process of Ethereum's transition to a proof of stake consensus mechanism |                                        Validators can push withdrawals from the Beacon chain to the EVM via a new "system-level" operation type                                         | shipped </br>EIP-4895[^2]  |
+| Upgrade                              | Track |    Topic    |                                             Description                                             |                                                                                     Expected effect                                                                                     | State of the art                                     |
+| :----------------------------------- | :---: | :---------: | :-------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------- |
+| Launch the Beacon Chain              |   -   |    Merge    |             A crucial step in Ethereum's shift to a proof of stake consensus mechanism              | Beacon Chain becomes the engine of block production, replacing mining. Validators have the role and responsibility for processing the validity of all transactions and proposing blocks | shipped </br> EIP-2982<span markdown='1'>[^1]</span> |
```

Comment:
Beacon chain was running indecently for almost 2 years. When it was launched, it didn't become an 'engine of production'. It was only listening to Ethereum and doing its own proof of stake with empty blocks. 

This part should say that Beacon chain is launched as an independent network connected to Ethereum, bootstraping validators and preparing for the merge